src/main/kotlin/no/nav/sykmeldinger/Environment.kt

```diff
@@ -18,11 +18,14 @@ data class Environment(
     val bekreftetTopic: String = "teamsykmelding.syfo-bekreftet-sykmelding",
     val sendtTopic: String = "teamsykmelding.syfo-sendt-sykmelding",
     val narmestelederLeesahTopic: String = "teamsykmelding.syfo-narmesteleder-leesah",
-    val cluster: String = getEnvVar("NAIS_CLUSTER_NAME"),
     val navnendringTopic: String = "pdl.leesah-v1",
     val schemaRegistryUrl: String = getEnvVar("KAFKA_SCHEMA_REGISTRY"),
     val kafkaSchemaRegistryUsername: String = getEnvVar("KAFKA_SCHEMA_REGISTRY_USER"),
-    val kafkaSchemaRegistryPassword: String = getEnvVar("KAFKA_SCHEMA_REGISTRY_PASSWORD")
+    val kafkaSchemaRegistryPassword: String = getEnvVar("KAFKA_SCHEMA_REGISTRY_PASSWORD"),
+    val historiskTopic: String = "teamsykmelding.sykmelding-historisk",
+    val behandlingsutfallConsumer: String = "teamsykmelding.sykmelding-behandlingsutfall",
+    val cluster: String = getEnvVar("NAIS_CLUSTER_NAME"),
+    val oldBehandlingsutfallTopic: String = "privat-syfo-sm2013-behandlingsUtfall",
```

Comment:
er ikke dette en header, ikke et topic?

Reply:
Ja, hva er dette? er det den gamle topicen fra onprem? Ble ikke den slettet?

Reply:
ja det er det gamle navnet for den topicen, det er det som ligger i header